src/Entity/Committee.php

```diff
@@ -368,4 +376,38 @@ public function getReportType(): string
     {
         return ReportType::COMMITTEE;
     }
+
+    public function getReferentTags(): Collection
+    {
+        return $this->referentTags;
+    }
+
+    public function addReferentTag(ReferentTag $referentTag): void
+    {
+        if (!$this->referentTags->contains($referentTag)) {
+            $this->referentTags->add($referentTag);
+        }
+    }
+
+    public function removeReferentTag(ReferentTag $referentTag): void
+    {
+        $this->referentTags->remove($referentTag);
+    }
+
+    public function removeReferentTags(): void
```

Comment:
maybe clearReferentTags could be better ? removeReferentTags should have an array of ReferentTag to specify each item we want remove no ?